pkg/util/ip_reservation_test.go

```diff
@@ -35,24 +35,30 @@ func TestValidateCIDR(t *testing.T) {
 		expected bool
 	}{
 		{
-			name:     "Valid /29 CIDR",
+			name:     "Valid /29 CIDR format",
 			cidr:     "192.168.92.192/29",
 			expected: true,
 		},
 		{
-			name:     "Invalid CIDR",
+			name:     "Invalid CIDR format",
 			cidr:     "192.168.92.192",
 			expected: false,
 		},
 		{
-			name:     "Valid /29 CIDR",
-			cidr:     "192.168.92.248/29",
-			expected: true,
-		}, {
-			name:     "Invalid CIDR with IP range overflow",
+			name:     "Invalid CIDR format with IP range overflow",
 			cidr:     "192.168.92.249/29",
 			expected: false,
 		},
+		{
+			name:     "Invalid CIDR format with network address greater than 29 bits",
+			cidr:     "192.168.92.249/30",
+			expected: false,
+		},
+		{
+			name:     "Valid CIDR format with IP network address less than 29 bits",
+			cidr:     "192.168.92.249/28",
+			expected: false,
```

Comment:
Since this is valid, expected should be true?

Reply:
was actually invalid as it was misaligned